### PR TITLE
Remove constraints

### PR DESCRIPTION
**Pull request by fabianrbz**
## Description

* load products from config file (used for url matching and sidenav generation)
* load oas definitions from config files
* load tutorials from the correct path

The only downside is that in `/coverage`, products will be mapped to the product list instead of the top level folders like we currently do. We can easily fix this.

### CODE REVIEW

**Comment by mheap** [comment]
@fabianrbz Let's fix the `/coverage` page in this PR too as it's an important page for the SDK team

**Comment by fabianrbz** [comment]
@mheap done

File: app/constraints/documentation_constraint.rb [hunk 1]
```diff
@@ -8,7 +8,12 @@ def self.product_with_parent
   end
 
   def self.product_with_parent_list
-    @product_with_parent_list ||= config.fetch('products', [])
+    byebug
+    if config['products'].nil? || config.fetch('products', []).size.zero?
```
Comment (fabianrbz) [inline app/constraints/documentation_constraint.rb]:
```suggestion
  if config.fetch('products', []).size.zero?
```

Comment (fabianrbz) [reply app/constraints/documentation_constraint.rb]: @benhayehudi can you remove the byebug?

Comment (benhayehudi) [reply app/constraints/documentation_constraint.rb]: 🤦‍♂ Will do now.

**Comment by fabianrbz** [comment]
@benhayehudi @mheap I'll leave this one to you to review/merge. i think it's in good shape.

**Comment by fabianrbz** [comment]
@benhayehudi @mheap I'll leave this one to you to review/merge. i think it's in good shape.

**Comment by benhayehudi** [comment]
> @benhayehudi @mheap I'll leave this one to you to review/merge. i think it's in good shape.

Since @mheap also is a reviewer on this, I'll wait for his ✅ before merging.

**Comment by mheap** [comment]
If you're happy with the changes, :shipit:

**Pull request merged**